tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/photo_collage.yaml

```diff
@@ -0,0 +1,25 @@
+context:
+  value:
+    title: We are 100% employee owned
+    intro: Our team's passion and expertise converge to deliver innovative solutions that empower your digital journey.
+    button_text: About us
+    url: '#'
+    images:
+      -
+      -
+      -
+      -
+      -
+      -
+
+tags:
+  srcset_image:
+    'image format-webp loading="lazy" fill-{408x405,816x810} sizes="(max-width: 598px) 816px, (min-width: 599px) 408px" class="photo-collage__image"':
+      raw: |
+        <img src="https://source.unsplash.com/408x405?people" class="photo-collage__image">
```

Comment:
To do: mock these properly as responsive images - this will be easier to do once we have some back-end code in place.